Packs/CiscoSMA/Integrations/CiscoSMA/CiscoSMA.yml

```diff
@@ -72,6 +72,12 @@ configuration:
   type: 0
   additionalinfo: Recipient filter value to fetch by message field.
   required: false
+- name: timeout
+  display: Timeout
+  type: 0
+  defaultvalue: '60'
+  additionalinfo: Timeout for the integration HTTP requests in seconds.
```

Comment:
```suggestion
  additionalinfo: HTTP requests timeout in seconds. The default is 60 seconds.
```

Reply:
I think no need to add the default as it has already been added.